src/main/flight/pid.c

```diff
@@ -445,8 +445,12 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     horizonFactorRatio = (100 - pidProfile->horizon_tilt_effect) * 0.01f;
     maxVelocity[FD_ROLL] = maxVelocity[FD_PITCH] = pidProfile->rateAccelLimit * 100 * dT;
     maxVelocity[FD_YAW] = pidProfile->yawRateAccelLimit * 100 * dT;
-    const float ITermWindupPoint = (float)pidProfile->itermWindupPointPercent / 100.0f;
-    ITermWindupPointInv = 1.0f / (1.0f - ITermWindupPoint);
+    float ITermWindupPoint = (float)pidProfile->itermWindupPointPercent / 100.0f;
+    if (ITermWindupPoint == 1.0f) {
```

Comment:
With `99` as max value in CLI this code seems unreachable

Reply:
thanks for picking up my mistake in removing const.  I hope I removed the correct float cast.